launcher/minecraft/mod/TexturePack.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include "Resource.h"
+
+#include <QImage>
+#include <QMutex>
+#include <QPixmap>
+#include <QPixmapCache>
+
+class Version;
+
+/* TODO:
+ *
+ * Store localized descriptions
+ * */
```

Comment:
I don't think texture packs have localized descriptions. At least it doesn't look like so from reading [the wiki page for it](https://minecraft.fandom.com/wiki/Texture_Pack#pack.mcmeta).